### DISCUSSION TITLE
CSRF for event's custom domain

### DISCUSSION THREAD

**Discussion by antropolouco** [pretix/pretix#4540] [Q&A]
Hey there!

I've set up a custom domain (subdomain in my case) in a dev install, but I'm getting a CSRF failure when trying to get a ticket on the event:

WARNING 2024-10-18 14:15:00,881 django.security.csrf log Forbidden (Origin checking failed - https://sub.domain.com does not match any trusted origins.): /cart/add
Forbidden (Origin checking failed - https://sub.domain.com does not match any trusted origins.): /cart/add
WARNING 2024-10-18 14:15:00,881 django.security.csrf log Forbidden (Origin checking failed - https://sub.domain.com does not match any trusted origins.): /cart/add
[18/Oct/2024 14:15:00] "POST /cart/add?next=/&next_error=/ HTTP/1.1" 403 1045

Proxy is all fine. What's the best way for authorizing such subdomain in Pretix, are there any vars I should use in the cfg file?

The custom redirects are automated, but the CSRF part seems to be not.

Thanks in advance for any responses.

**Comment by raphaelm** [comment]
This is because you are using the dev setup in a way it should not be used, see other threads.

**Reply by antropolouco** [reply]
I know it shouldn't be used in production, but...

Is it simply impossible to set domains in CSRF configuration using a dev install, then?

Thanks

**Reply by raphaelm** [reply]
No, it is probably not impossible, but I don't know what you exactly need to do and I won't spend the time to find out, if there is a perfectly reasonable recommended way of installing ;)

**Reply by antropolouco** [reply]
Thank you for not spending your time, Rapha!

**Reply by raphaelm** [reply]
You're welcome!
